src/PostgrestClient.ts

```diff
@@ -21,7 +21,8 @@ export default class PostgrestClient<
     : string & keyof Database,
   Schema extends GenericSchema = Database[SchemaName] extends GenericSchema
     ? Database[SchemaName]
-    : any
+    : any,
+  TypesVersion extends 'next' | undefined = undefined
```

Comment:
Here is our feature flag, it needs to be drilled down to `PostgrestQueryBuilder` and  `Filter` but that's the best way I've found to implement it. Open to suggestions tough.